common/index.ts

```diff
@@ -31,6 +31,13 @@ export const ERROR_MISSING_ROLE_PATH = '/missing-role';
 
 export const MAX_INTEGER = 2147483647;
 
+export const GLOBAL_TENANT_SYMBOL = '';
```

Comment:
This file has some of the changes from https://github.com/opensearch-project/security-dashboards-plugin/pull/1184 can we include all of them?

Reply:
I've only included the ones that were required. We can backport #1184 to 1.x and 1.3 if required.